resources/apt-packages.txt

```diff
@@ -0,0 +1,38 @@
+# utilities
+git
+mercurial
+pkg-config
+wget
+# debian-provided interpreters
+python-pip
+python2.7
+python2.7-dev
```

Comment:
Do you mind punting this until we can take advantage of cloud build and do a two stage build?